gulp-tasks/build-packages.js

```diff
@@ -1,204 +1,59 @@
-  if (!fs.pathExistsSync(moduleIndexPath)) {
-    logHelper.error(ERROR_NO_MODULE_INDEX + packageName);
-    return Promise.reject(ERROR_NO_MODULE_INDEX + packageName);
-  }
-
-  const pkgJson = require(path.join(packagePath, 'package.json'));
-  if (!pkgJson.workbox || !pkgJson.workbox.browserNamespace) {
-    logHelper.error(ERROR_NO_NAMSPACE + ' ' + packageName);
-    return Promise.reject(ERROR_NO_NAMSPACE + ' ' + packageName);
-  }
-
-  const namespace =
-    `${constants.NAMESPACE_PREFIX}.${pkgJson.workbox.browserNamespace}`;
-  const outputFilename = `${packageName}.${buildType.slice(0, 4)}.js`;
-  const outputDirectory = path.join(packagePath,
-    constants.PACKAGE_BUILD_DIRNAME, constants.BROWSER_BUILD_DIRNAME);
-
-  logHelper.log(oneLine`
-    Building Browser Bundle for
-    ${logHelper.highlight(packageName)}.
-  `);
-  logHelper.log(`    Namespace: ${logHelper.highlight(namespace)}`);
-  logHelper.log(`    Filename: ${logHelper.highlight(outputFilename)}`);
-
-  return rollupStream({
-    input: moduleIndexPath,
-    rollup,
-    format: 'iife',
-    exports: 'named',
-    name: namespace,
-    sourcemap: true,
-    globals,
-    external: externalAndPure,
-    pureExternalModules: externalAndPure,
-    plugins: rollupHelper.getDefaultPlugins(buildType),
-    onwarn: (warning) => {
-      if (buildType === 'production' &&
-        warning.code === 'UNUSED_EXTERNAL_IMPORT') {
-        // This can occur when using rollup-plugin-replace.
-        logHelper.warn(`[${warning.code}] ${warning.message}`);
-        return;
-      }
-
-      // The final builds should have no warnings.
-      throw new Error(`Unhandled Rollup Warning: [${warning.code}] ` +
-        `'${warning.message}'`);
-    },
-  })
-  .on('error', (err) => {
-    const args = [];
-    Object.keys(err).forEach((key) => {
-      args.push(`${key}: ${err[key]}`);
-    });
-    logHelper.error(err, `\n\n${args.join('\n')}`);
-    throw err;
-  })
-  // We must give the generated stream the same name as the entry file
-  // for the sourcemaps to work correctly
-  .pipe(source(moduleIndexPath))
-  // gulp-sourcemaps don't work with streams so we need
-  .pipe(buffer())
-  // This tells gulp-sourcemaps to load the inline sourcemap
-  .pipe(sourcemaps.init({loadMaps: true}))
-  // This renames the output file
-  .pipe(rename(outputFilename))
-  // This writes the sourcemap alongside the final build file
-  .pipe(sourcemaps.write('.'))
-  .pipe(gulp.dest(outputDirectory));
-};
-
-const cleanPackages = (packagePath) => {
+const cleanPackage = (packagePath) => {
   const outputDirectory = path.join(packagePath,
-    constants.PACKAGE_BUILD_DIRNAME, constants.BROWSER_BUILD_DIRNAME);
-  return fs.remove(outputDirectory);
+    constants.PACKAGE_BUILD_DIRNAME);
+  return fse.remove(outputDirectory);
 };
 
 gulp.task('build-packages:clean', gulp.series(
-  packageRunnner('build-packages:clean', cleanPackages)
+    packageRunnner.wrapFunction(
+      'Cleaning Package',
```

Comment:
I've been trying to follow the pattern of using the gulp task name so it's clear from logs where this log is from: in this case ''build-packages:clean''